docs/sources/monitor/monitor-docker-containers.md

```diff
@@ -0,0 +1,221 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/tutorials/scenarios/monitor-docker-containers/
+description: Learn how to use Grafana Alloy to monitor Docker containers
+menuTitle: Monitor Docker
+title: Monitor Docker containers with Grafana Alloy
+weight: 200
+---
+
+# Monitor Docker containers with {{% param "FULL_PRODUCT_NAME" %}}
+
+Docker containers provide statistics and logs.
+You can use the `docker stats` and `docker logs` commands to show the metrics and logs but this just shows the output in a terminal and it's a fixed snapshot in time.
+If you use {{< param "PRODUCT_NAME" >}} to collect the metrics and logs and forward them to a Grafana stack, you can create a Grafana dashboard to monitor your Docker container.
+
+## Before you begin
+
+This example requires:
+
+* Docker
+* Git
+
+{{< admonition type="note" >}}
+The `docker` commands require administrator privileges.
+{{< /admonition >}}
```

Comment:
We're still talking through this in our Slack thread, but there may be something worth noting here about some os-specific adjustments needed. For me, this relates to host and/or VM permissions that are needed by the Alloy VM (e.g. something to do with cgroup). I think this might be specific to this particular demo, which needs that lower level of access to Docker, but I'll confirm that as I look through the other ones!